doc/whatsnew.rst

```diff
@@ -19,7 +19,9 @@ v0.3.1.dev
 - Enhance AJD: add ``init`` to :func:`pyriemann.utils.ajd.ajd_pham` and :func:`pyriemann.utils.ajd.rjd`,
   add ``warm_restart`` to :class:`pyriemann.spatialfilters.AJDC`. :pr:`196` by :user:`qbarthelemy`
 
-- Add parameter ``sampling_method`` to :func:`pyriemann.dataset.sample_gaussian_spd`, with ``rejection`` accelerating 2x2 matrices generation. :pr:`198` by :user:`Artim436`
+- Add parameter ``sampling_method`` to :func:`pyriemann.datasets.sample_gaussian_spd`, with ``rejection`` accelerating 2x2 matrices generation. :pr:`198` by :user:`Artim436`
+
+- Add geometric medians for Euclidean and Riemannian metrics: :func:`pyriemann.utils.median.median_euclid` and :func:`pyriemann.utils.median.median_riemann`. :pr:`200` by :user:`qbarthelemy`
```

Comment:
I am not a fan the repetition of median in the import name 

would you object to :

:func:`pyriemann.utils.median_riemann`

or you think that utils module is already too crowded?

Reply:
Do you suggest to add these imports in `utils.__init__` file?
```
from .mean import (
    mean_covariance,
    mean_ale,
    mean_alm,
    mean_euclid,
    mean_harmonic,
    mean_identity,
    mean_kullback_sym,
    mean_logdet,
    mean_logeuclid,
    mean_power,
    mean_riemann,
    mean_wasserstein,
    maskedmean_riemann,
    nanmean_riemann,
)
from .median import (
    median_euclid,
    median_riemann,
)
```

Reply:
> Yes. What do you think ?

Ok for me!